docs/common/craft-parts/craft-parts.wordlist.txt

```diff
@@ -24,6 +24,9 @@ BeforeValidator
 BuildPackageNotFound
 BuildPackagesNotInstalled
 BuildState
+CargoPackagePlugin
+CargoPackagePluginEnvironmentValidator
+CargoPackagePluginProperties
```

Comment:
```suggestion
```

Reply:
Yep, thank you! Double-checked and couldn't find any other leftovers.